cg-spec/editors_draft.html

```diff
@@ -941,7 +941,206 @@ <h2>SPARQL-star Query Results XML Format</h2>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          INSERT {
+            GRAPH :graph2 { ?s ?p ?o }
+          }
+          WHERE {
+            { <<?s ?p ?o>> ?pp ?oo }
+            UNION
+            { ?ss ?pp <<?s ?p ?o>> }
+          }
+          -->
+        </pre>
+
+      </section>
+
+    </section>
+
+<!--
+    <section>
+      <h2>Initial Definitions</h2>
+
+      <p>We begin by introducing an RDF-star-aware notion of a <a data-cite="SPARQL11-UPDATE#defn_graphStore">graph store</a> [[SPARQL11-UPDATE]].</p>
+
+      <p>A <dfn data-lt="graphStore-star">RDF-star graph store</dfn> |GS|&nbsp;=&nbsp;{ |DG|, (|iri|<sub>1</sub>, |G|<sub>1</sub>),  ... , (|iri|<sub>|n|</sub>, |G|<sub>|n|</sub>) } is a mutable container of <a>RDF-star graphs</a>. It has one unnamed (default) slot and zero or more named slots. The unnamed slot holds an RDF-star graph&nbsp;|DG|, which is called the <em>default graph</em>; each named slot is a pair (|iri|<sub>|i|</sub>, |G|<sub>|i|</sub>) of an RDF-star graph&nbsp;|G|<sub>|i|</sub> and an associated <a>IRI</a>&nbsp;|iri|<sub>i</sub> where</p>
+      <ul>
+        <li>|n| ≥ 0,</li>
+        <li>for each 1 ≤ |i| ≤ |n|, |G|<sub>|i|</sub> is an RDF-star graph associated with the named slot identified by IRI&nbsp;|iri|<sub>|i|</sub>,</li>
+        <li>all these IRIs are distinct, i.e., |i| ≠ |j| implies that |iri|<sub>|i|</sub> ≠ |iri|<sub>|j|</sub>.</li>
+      </ul>
+
+    </section>
+-->
+
+    <section>
+      <h2>Grammar</h2>
+
+      <p>SPARQL-star Update is an extension of the SPARQL Update language [[SPARQL11-UPDATE]]. As mentioned in <a data-cite="SPARQL11-UPDATE#grammar">SPARQL&nbsp;1.1 Update, Appendix&nbsp;C</a>&nbsp;[[SPARQL11-UPDATE]], the grammar of the latter is provided as part of the <a data-cite="SPARQL11-QUERY#sparrqlGrammar">SPARQL&nbsp;1.1 Query grammar</a>. Similarly, the grammar of SPARQL-star Update is provided as part of the grammar of the SPARQL-star query language, which is defined by the SPARQL&nbsp;1.1 Query grammar with the extensions specified in <a href="#sparql-star-grammar"></a>. As a result of these extensions, the production rules <a data-cite="SPARQL11-QUERY#rQuadData">QuadData</a> and <a data-cite="SPARQL11-QUERY#rQuadPattern">QuadPattern</a>, which are used in the definition of SPARQL Update [[SPARQL11-UPDATE]], are extended to capture nested triples and nested triple patterns as demonstrated in the examples above.</p>
```

Comment:
s/sparrqlGrammar/sparqlGrammar ?

Reply:
Nice catch. Thanks!
Commit https://github.com/w3c/rdf-star/pull/98/commits/817f3746d3167bc00becb6a7ba60b5e076313484 fixes it.